### ISSUE TITLE
[Request] Add support for -UseBasicParsing

### ISSUE THREAD

**Issue by BudLightning** [iRon7/Read-HtmlTable#7]
Hopefully this is still monitored and/or receiving updates. I'd like to request the ability to flag -UseBasicParsing for the invoke-webrequest call, for cases such as using windows server core

**Comment by iRon7** [comment]
@BudLightning 

> the ability to flag -UseBasicParsing

Yes, I still monitor and maintain this script but I don't actually understand where you ask for. When you supply a `uri` for the `InputObject` (or use `uri` parameter), only the `raw` content of the results of `invoke-webrequest` is used (line `150` of the `.ps1` script). Saying that, you might even do your own `invoke-webrequest` and pass the html content to this `ConvertFrom-HtmlTable` cmdlet.
In other words, please elaborate, where exactly are you look for, what are you trying to prevent?

**Comment by BudLightning** [comment]
I'm guessing I'm just being a bit dumb lol, my bad. For whatever reason it hadn't occurred to me to invoke-webrequest with -usebasicparsing and pass that to the read-htmltable function.. I apologize and thank you for your time!
